docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,90 @@
 
-The full list of HSA system architecture platform requirements are here:
-`HSA Sys Arch Features <http://hsafoundation.com/wp-content/uploads/2021/02/HSA-SysArch-1.2.pdf>`_.
+Platform atomics are the set of atomic operations that performs Read Modify Write (RMW) actions across multiple processor, devices, and memory location synchronously without any interruption. Platform atomics used in ROCm are:
 
-AMD ROCm Software uses the new PCI Express 3.0 (Peripheral Component Interconnect Express [PCIe]
-3.0) features for atomic read-modify-write transactions which extends inter-processor synchronization
-mechanisms to IO to support the defined set of HSA capabilities needed for queuing and signaling
-memory operations.
+* Update HSA queue's ``read_dispatch_id``: 64 bit atomic add used by the command processor on the
+  GPU agent. It updates the packet ID it processed.
+* Update HSA queue's ``write_dispatch_id``: 64 bit atomic add used by the CPU and GPU agent. It supports multi-writer queue insertions.
+* Update HSA Signals: 64 bit atomic operation used for CPU & GPU synchronization.
 
-The new PCIe atomic operations operate as completers for ``CAS`` (Compare and Swap), ``FetchADD``,
-``SWAP`` atomics. The atomic operations are initiated by the I/O device which support 32-bit, 64-bit and
-128-bit operand which target address have to be naturally aligned to operation sizes.
+Atomic operations
+-------------------
+An atomic operation is a sequence of computing instructions that are executed as a single, indivisible unit. These instructions needs to be completed at its entirety without any interruption or should not execute at all. It is a non-posted transaction supporting 32-bit and 64-bit address formats. A successful completion of these operations are identified by response of completion containing the operation result. However, any errors associated with the operation like issue in accessing the target location or executing the atomic operation are signaled to the requester by updating the Completion Status field present in the Completion Descriptor. Depending upon the error, the Completion Status field is updated to Completer Abort (CA) or Unsupported Request (UR).
 
-For ROCm the Platform atomics are used in ROCm in the following ways:
+PCIe in ROCm
+======================
+AMD ROCm uses atomic RMW transactions that extends inter-processor synchronization mechanisms to Input/Output (I/O) devices starting from  PCIe 3.0. It supports the defined set of HSA capabilities needed for queuing and signaling memory operations. 
 
-  * Update HSA queue's read_dispatch_id: 64 bit atomic add used by the command processor on the
-    GPU agent to update the packet ID it processed.
-  * Update HSA queue's write_dispatch_id: 64 bit atomic add used by the CPU and GPU agent to
-    support multi-writer queue insertions.
-  * Update HSA Signals -- 64bit atomic ops are used for CPU & GPU synchronization.
+PCIe operates as a completer for atomic operations like:  
```

Comment:
```suggestion
PCIe operates as a completer for atomic operations such as:  
```

this is a style thing

Reply:
Rewrote as a paragraph. So no longer valid.